project.clj

```diff
@@ -28,12 +28,13 @@
                                    [org.slf4j/slf4j-simple "1.7.30"]
                                    [com.cognitect/transit-clj "1.0.324"]
                                    [spootnik/signal "0.2.4"]
-                                   [me.mourjo/dynamic-redef "0.1.0"]]}
-             :lein-to-deps {:source-paths ["deps"]}
-             ;; This is for self-generating certs for testing ONLY:
-             :test {:dependencies [[org.bouncycastle/bcprov-jdk15on "1.69"]
+                                   [me.mourjo/dynamic-redef "0.1.0"]
+                                   ;; This is for self-generating certs for testing ONLY:
+                                   [org.bouncycastle/bcprov-jdk15on "1.69"]
```

Comment:
```suggestion
                                   [org.bouncycastle/bcprov-jdk18on "1.72"]
```

How about this while we're at it?

Reply:
We're not supporting anything below 1.8, so the upgrade should be good on that front.

Reply:
https://github.com/clj-commons/aleph/pull/634/commits/82a39b815d9aac4bcfe6375a83fdded7895c0b57